lib/CPathfinder.h

```diff
@@ -21,6 +23,19 @@ struct TerrainTile;
 
 struct DLL_LINKAGE CGPathNode
 {
+	typedef EPathfindingLayer ELayer;
+
+	enum ENodeAction
```

Comment:
ENodeAction need re-design. When I implemented it I'm completely missed fact that it's possible to have both disembark and battle. Also battle themself can occur on both visit and blocking visit tiles, etc.

So I still thinking about elegant solution, but only way I see currently is bitwise operations or using struct.